_rules/SC1-2-1-video-only-element-transcript.md

```diff
@@ -0,0 +1,129 @@
+---
+name: video only element has transcript
+description: |
+  Non-streaming `video` elements without audio must have all visual information available in a transcript.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+- Visual output
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video doesn't contain audio.
+
+### Expectation 1
+
+A text transcript is available for the test target, either on the page or available through a link.
+
+### Expectation 2
+
+The text transcript (from expectation 1) contains all the visual information of the test target.
+
+**Note**: A "text transcript" in the context of this rule is defined in WCAG 2 as a [alternative for time based media](https://www.w3.org/TR/WCAG21/#dfn-alternative-for-time-based-media).
```

Comment:
Minor: for "poster" we link directly to the HTML5 spec, whereas for "text transcript" we add this note that makes the connection to the source spec (WCAG in this case). Should we promote one style for consistency? Does/should the style guide say anything about that?